### DISCUSSION TITLE
时钟闪烁

### DISCUSSION THREAD

**Discussion by Duguwochou** [martinet101/ElevenClock#849] [Q&A]
当鼠标放在任务栏图标上时，时钟闪烁。我使用的是Windows11 Dev 25163版。

**Comment by marticliment** [comment]
Hi, please post in english so we all can understand what you want to say

**Reply by win98se** [reply]
I'll translate his issue statement - “When mouse cursor is placed on the taskbar icon(s), the clock blinks/flashes.” He also mentions that he uses Windows 11 Insider Preview build 25163 (Dev channel).
